versions.cmake

```diff
@@ -130,23 +130,22 @@ else()
 endif()
 
 if (WIN32)
+  set(_itk_sha512 "f53724a3f7eb8758aded361257ec7d581e4efa0e45ee138e4b6a307b4fc2a980100fdc12c82f9ee14ed539459f2f5078a696292e1dba352edbc595b5ac4d85c3")
   add_revision(itk
-    URL "http://www.tomviz.org/files/itk-4.11.0-windows-64bit.zip"
-    URL_MD5 "8357ebd1ccce7d2366a96c569e2b3d11")
+    URL "http://www.tomviz.org/files/itk-4.12.0-windows-64bit.zip"
+    URL_HASH SHA512=${_itk_sha512})
   add_revision(fftw
     URL "http://www.tomviz.org/files/fftw-3.3.4-windows-64bit.zip"
     URL_MD5 "90ca2a2cd26c62bc85b11ec7f165e716")
 elseif(APPLE)
-  set(_itk_sha512 "00b2073761fb6d456bbaf8883a0362a56c2a49228441408e82d4293b1633072fbcba8b0bed73fb3633c86467352d74dea479c4330098d79316e15e1a2bf3b95a")
+  set(_itk_sha512 "ecf7e98564d8b9291df1d72a4169ec84e4dd5b5f605cb8d8d1ca6983a1c2b0abcbff5f9c468fe5cf59724903ce0d74f07840062961e2d5728dcd3268fc22f5f6")
```

Comment:
Unfortunately it looks like it was built with Python 2.  I'm rebuilding it now.  It was missing a few variables to make it use the python from the tomviz superbuild too.  I'm guessing it didn't get updated last time we built ITK and was out of date.